### PR TITLE
Make login and signup function

### PR DESCRIPTION
**Pull request by Tomoki108**
***Outline***
- Login and signup (actual name: register) functions added. 

***Commands to run***
- none

***Preconditions***
- PR#2 and PR#3 are already merged

***Expected outputs***
- login and signup view, controller and middleware are created.
- login and signup functions work properly on browser. 

***asana***
- https://app.asana.com/0/984197702640917/984308420011949

### CODE REVIEW

**Comment by jeremiahC** [comment]
Please fix commits, i think commits are not for this branch please push only the original commit of this Pull request

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,4 @@ Homestead.yaml
 npm-debug.log
 yarn-error.log
 
-/public/css
```
Comment (jeremiahC) [inline .gitignore:13]: why delete this one?

File: app/Http/Controllers/Auth/RegisterController.php [hunk 2]
```diff
@@ -66,6 +66,7 @@ protected function create(array $data)
         return User::create([
             'name' => $data['name'],
             'email' => $data['email'],
+            'avatar'=> $data['email'],
```
Comment (jeremiahC) [inline app/Http/Controllers/Auth/RegisterController.php]: why is the value of avatar variable email?

**Review by jeremiahC — CHANGES_REQUESTED** [review]
Please use sass instead of css, reference link is this one https://sass-lang.com/documentation/file.SASS_REFERENCE.html

File: app/User.php [hunk 3]
```diff
@@ -16,7 +16,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'name', 'email', 'avatar', 'admin', 'password',
```
Comment (jeremiahC) [inline app/User.php]: remove trailing space (extra space) at the end

File: .gitignore (hunk 1 above)
Comment (Tomoki108) [reply .gitignore:13]: Because I want to use default css file.

File: app/Http/Controllers/Auth/RegisterController.php (hunk 2 above)
Comment (Tomoki108) [reply app/Http/Controllers/Auth/RegisterController.php]: It's mistake and I'll fix it. (And as additional info, user can set his avatar in profile edit page not in signup page in my plan.)

File: resources/views/welcome.blade.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>Laravel</title>
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet" type="text/css">
+
+        <!-- Styles -->
+        <style>
+            html, body {
```
Comment (jeremiahC) [inline resources/views/welcome.blade.php]: please transfer this internal css code to a externals css file

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -0,0 +1,73 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Login') }}</div>
```
Comment (jeremiahC) [inline resources/views/auth/login.blade.php]: what is the purpose of using `__('Login')` compared to just using normal text `Login`.

Comment (Tomoki108) [reply resources/views/auth/login.blade.php]: It's made like this by (php artisan make:auth). Should I modify it?

File: resources/views/welcome.blade.php (hunk 4 above)
Comment (Tomoki108) [reply resources/views/welcome.blade.php]: This welcome.blade.php is temporary top page which is made by (php artisan make:auth). I'll erase this page and make my own top page in near future. So, please ignore the problem.

File: resources/views/auth/login.blade.php (hunk 5 above)
Comment (jeremiahC) [reply resources/views/auth/login.blade.php]: better to change it

Comment (jeremiahC) [reply resources/views/auth/login.blade.php]: also others that are using this function

File: resources/views/welcome.blade.php (hunk 4 above)
Comment (jeremiahC) [reply resources/views/welcome.blade.php]: better if erased now if no use.

File: resources/views/auth/login.blade.php (hunk 5 above)
Comment (Tomoki108) [reply resources/views/auth/login.blade.php]: Okay, I got it, I'll change it and push.

File: resources/views/welcome.blade.php (hunk 4 above)
Comment (Tomoki108) [reply resources/views/welcome.blade.php]: Okay, I got it, I'll change it and push.

File: resources/views/layouts/app.blade.php [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+</head>
+<body>
+    <div id="app">
+        <nav class="navbar navbar-expand-md navbar-light navbar-laravel">
+            <div class="container">
+                <a class="navbar-brand" href="{{ url('/') }}">
+                    {{ config('app.name', 'Laravel') }}
+                </a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="{{ __('Toggle navigation') }}">
```
Comment (jeremiahC) [inline resources/views/layouts/app.blade.php:29]: this line is not changed, please change

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**